### ISSUE TITLE
Shift OpenMainMenuOnHover to UserPreferences

### ISSUE THREAD

**Issue by stefanhaerter** [RotherOSS/otobo#2539]
It is great to have OpenMainMenuOnHover as a sysconfig option, but I would appreciate it even more to have it as UserPreference. I would suggest to keep the sysconfig option as default, but to add a UserPreference which can overrule it.

**Comment by StefanAbel-OTOBO** [comment]
that would be nice!

**Comment by stefanhaerter** [comment]
As described in https://github.com/RotherOSS/otobo/pull/2574#issuecomment-1772206402, there already exists a working, but currently not displayed mechanism within user preferences to handle user modifications of sysconfig settings. It should be discussed if this mechanism is to be made available (again?) and if so, I'd love to add OpenMainMenuOnHover into the list of user-modifiable sysconfig options.

**Comment by bschmalhofer** [comment]
I second that the user config mechanism should be reactivated. If this is done then the list of  user-modifiable sysconfig options should be reviewed too.

**Comment by bschmalhofer** [comment]
See also #1788.

**Comment by stefanhaerter** [comment]
Provided the change for the original topic in https://github.com/RotherOSS/otobo/pull/4404.

The bigger topic of making user modifications available again is tackled in #1788.

Edit: The PR is based on rel-11_0, as I consider the change to have no effect because of the user modifications of sysconfigs are not active frontend-wise, but I consider this a nice preparation in case this task is taken on somewhen.

**Comment by gyarakilaszlo** [comment]
Here is a proposal to solve this issue: #5224

**Comment by stefanhaerter** [comment]
The comment is related to #1788, isn't it?

**Comment by gyarakilaszlo** [comment]
Not exactly. The #1788 proposal simply enables the feature that allows users to replace some system settings with their own values. Proposal #5224 contains controls for more settings, including OpenMainMenuOnHover, which are found in user preferences.
